src/incubator/TextField/__tests__/index.spec.js

```diff
@@ -0,0 +1,78 @@
+import React, {useState} from 'react';
+import {fireEvent, render} from '@testing-library/react-native';
+import TextField from '../index';
+
+describe('TextField', () => {
+  describe('hint prop', () => {
+    const props = {
+      testID: 'field',
+      placeholder: 'Placeholder',
+      hint: 'Hint'
+    };
+
+    it('should hint text replace placeholder when input is focused', () => {
```

Comment:
Consider adding the same test but when the user doesn't pass `hint`

Reply:
Done